### PR TITLE
Not-found-animation + Translating-animation + text #18 #20

### PR DESCRIPTION
**Pull request by ArturLabenskyy**
I finished 2 components for "Translating text" and "Not Found" with text for each one.

### CODE REVIEW

**Comment by ArturLabenskyy** [comment]
got 62 conflicts at package-lock.json 😥

File: client/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "npm": "^9.6.6",
```
Comment (yishain11) [inline client/package.json:14]: remove npm

**Review by yishain11 — CHANGES_REQUESTED** [review]

File: client/src/App.css [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+    max-width: 1280px;
+    margin: 0 auto;
+    padding: 2rem;
+    text-align: center;
 }
 
 .logo {
```
[the remark this answers is not in the document]
Comment (ArturLabenskyy) [reply client/src/App.css:8]: On the main branch, in App.jsx file there no import for App.css

**Comment by ArturLabenskyy** [comment]
I made refactoring to Animation component so it possible to reuse it for both animations with text: NotFoundAnimation and TranslatingAnimation

File: client/src/components/Animation/Animation.styled.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            line-height: 133%;
+            color: #333333;
+        }
+    }
+
+    // iPad + Tablet screen
+    @media only screen and (min-width: 481px) and (max-width: 768px) {
+    }
+
+    // Small screen + Laptop screen
```
Comment (yishain11) [inline client/src/components/Animation/Animation.styled.js:32]: remove - we work obly with mobile

File: client/src/components/Animation/Animation.styled.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    @media only screen and (min-width: 769px) and (max-width: 1024px) {
+    }
+
+    //  Desktops + large screen
```
Comment (yishain11) [inline client/src/components/Animation/Animation.styled.js:36]: remove - we work obly with mobile

File: client/src/components/Animation/Animation.styled.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    @media only screen and (min-width: 1025px) and (max-width: 1200px) {
+    }
+
+    // Extra large + TV screen
```
Comment (yishain11) [inline client/src/components/Animation/Animation.styled.js:40]: remove - we work obly with mobile

File: client/src/App.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (yishain11) [reply client/src/App.css:14]: its up to you and toni - if you use app.css - remove all the unrelevant classes and animations.
if you don't need it - delete the file and remove the import
currnlty you have a mix of both - vite classes and your styling

**Review by yishain11 — CHANGES_REQUESTED** [review]

File: client/src/App.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (ArturLabenskyy) [reply client/src/App.css]: App.css changed

**Comment by ArturLabenskyy** [comment]
Conflicts resolved

**Review by yishain11 — CHANGES_REQUESTED** [review]
fix home page. in main it looks ok - with the headr at the top, and in your branch its messed

**Comment by ArturLabenskyy** [comment]
Waiting for approve

**Review by yishain11 — CHANGES_REQUESTED** [review]
header is still bad in your branch

File: client/package.json [hunk 6]
```diff
@@ -1,33 +1,34 @@
...
+    "type": "module",
+    "scripts": {
+        "dev": "vite",
+        "build": "vite build",
+        "lint": "eslint src --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+        "preview": "vite preview"
+    },
+    "dependencies": {
+        "@reduxjs/toolkit": "^1.9.5",
+        "parse-conflict-json": "^3.0.1",
```
Comment (yishain11) [inline client/package.json:14]: are you using this?
if yes - should it be as dependancy or as dev dependancy?

File: client/src/App.css [hunk 7]
```diff
@@ -1,44 +1,23 @@
...
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  a:nth-of-type(2) .logo {
-    animation: logo-spin infinite 20s linear;
-  }
+#root {
+    max-width: 1280px;
+    margin: 0 auto;
+    padding: 2rem;
```
Comment (yishain11) [inline client/src/App.css:10]: this messes the style in other pages

**Review by yishain11 — CHANGES_REQUESTED** [review]

File: client/src/App.css (hunk 7 above)
Comment (ArturLabenskyy) [reply client/src/App.css]: Fixed

File: client/src/components/Tagline/Tagline.styled.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
+import { styled } from 'styled-components'
```
Comment (yishain11) [inline client/src/components/Tagline/Tagline.styled.js:1]: this cuases an error. fix it

**Review by yishain11 — CHANGES_REQUESTED** [review]

**Pull request merged**